components/trades/TradeComposer.tsx

```
@@ -0,0 +1,445 @@
+                >
+                    <div className="flex h-auto min-h-[40px] items-center gap-2 border-zinc-800/50 border-b px-4 py-1.5 flex-wrap">
+                        <div className="text-sm text-zinc-500 shrink-0">Send trade to:</div>
+                        <div className="flex flex-1 items-center gap-2 flex-wrap min-w-[200px]">
+                            {selectedRecipients.map(recipient => (
+                                <div key={recipient.name} className="flex items-center gap-1.5 rounded-full border border-zinc-700 py-0.5 pr-1.5 pl-1 bg-zinc-800">
+                                    {/* {recipient.image && <Image src={recipient.image} alt={recipient.name} width={20} height={20} className="rounded-full" />} */}
+                                    <div className={cn("flex h-5 w-5 items-center justify-center rounded-full text-[10px] font-medium text-white", getAvatarColorClass(recipient.name))}>
+                                        {recipient.name.charAt(0).toUpperCase()}
+                                    </div>
+                                    <div className="text-sm text-white">{recipient.name}</div>
+                                    <button onClick={() => handleRemoveRecipient(recipient.name)} className="text-zinc-400 hover:text-white">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)